config/initializers/cors.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
```

Comment:
- [ ] In your README file, you should include a detailed step of how one can set up the app on their local machine. You missed out the following: 
    - rails db:create
    - rails db:migrate
    - rails db:seed
    - rails server

- [ ] After setting up your app, I lunch it. this is what I see 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/218265745-2d30b997-0d91-4dc8-a49e-3896b6ea05b1.png)

- [ ] Also, you should include in your README the JSON path e.g (/users/:user_id/posts) that you added in this milestone. 
- [ ] I am not able to create a User. Kindly check your code. Because of this I am not able to test your endpoints.

Please fix all this.